.circleci/config.yml

```diff
@@ -24,10 +24,8 @@ jobs:
           set -e
           echo "Cloning ${ST2_PACKAGES_BRANCH:-master} branch of st2-packages"
           git clone --branch ${ST2_PACKAGES_BRANCH:-master} --depth 1 ${ST2_PACKAGES_REPO} ~/st2-packages
-          # Use DEPRECATED/all-in-one for now, we'll have to circle back around
-          # and fix this to use the master branch
-          echo "Cloning ${ST2_DOCKER_BRANCH:-DEPRECATED/all-in-one} branch of st2-docker"
-          git clone --branch ${ST2_DOCKER_BRANCH:-DEPRECATED/all-in-one} --depth 1 ${ST2_TEST_ENVIRONMENT} ~/st2-docker
+          echo "Cloning ${ST2_DOCKER_BRANCH:-master} branch of st2-docker"
+          git clone --branch ${ST2_DOCKER_BRANCH:-master} --depth 1 ${ST2_TEST_ENVIRONMENT} ~/st2-docker
           make -C ~/st2-docker env
```

Comment:
Oh. Apparently this target is missing on the master branch. That's why CI is failing.